### ISSUE TITLE
先执行 tab_endAnimation，导致collectionView不能触摸滚动

### ISSUE THREAD

**Issue by chouson** [tigerAndBull/TABAnimated#129]
### New Issue Checklist

* [x] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
是否使用其他框架，管理scrollEnabled属性

**Comment by tigerAndBull** [comment]
是否尝试过2.5.1版本

**Comment by chouson** [comment]
版本是2.5.0的，没有使用其他框架，都是用系统原生API，如果按照顺序执行 tab_startAnimation -> tab_endAnimation 就没有问题，如果先执行 tab_endAnimation，再执行 tab_startAnimation -> tab_endAnimation，collectionView就不能触摸
